### PR TITLE
i18n(zh-cn): translate `guides/cms/keystonejs.mdx`

### PR DESCRIPTION
**Pull request by miroim**
#### What kind of changes does this PR include?

- Translated content

#### Description

translate `\guides\cms\keystonejs.mdx`

### CODE REVIEW

**Comment by miroim** [comment]
cc: @liruifengv

File: src/content/docs/zh-cn/guides/cms/keystonejs.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: KeystoneJS & Astro
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/keystonejs.mdx:2]:
```suggestion
title: KeystoneJS 与 Astro
```

File: src/content/docs/zh-cn/guides/cms/keystonejs.mdx [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+description: 使用 KeystoneJS 作为 CMS 将內容添加到你的 Astro 项目
+type: cms
+stub: true
+service: KeystoneJS
+i18nReady: true
+---
+
+[KeystoneJS](https://keystonejs.com/) 是一个开源无头 CMS（Content Management System，内容管理系统）。只需描述你的数据结构，即可使用 KeystoneJS 的 GraphQL API 和数据管理界面。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/keystonejs.mdx:10]:
```suggestion
[KeystoneJS](https://keystonejs.com/) 是一个开源无头的 CMS（Content Management System，内容管理系统）。只需描述你的数据结构，即可使用 KeystoneJS 的 GraphQL API 和数据管理界面。
```

**Review by 100gle — CHANGES_REQUESTED** [review]

**Comment by 100gle** [comment]
LGTM

**Review by 100gle — APPROVED** [review]

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @miroim 🙌

**Pull request merged**